src/main/resources/tags/climbing_pick_strong_surfaces.json

```diff
@@ -4,6 +4,11 @@
 		"#slimefun:terracotta",
 		"#slimefun:stone_variants",
 		"minecraft:netherrack",
+		"minecraft:calcite",
+		"minecraft:deepslate",
+		"minecraft:dripstone_block",
+		"minecraft:smooth_basalt",
+		"minecraft:tuff",
```

Comment:
This will break all versions below 1.18.
see how blackstone is added.
Change them all to that.